### PR TITLE
Project: Business Site - Sprinkle

### PR DESCRIPTION
**Pull request by Rosanna86**
My page is called Sprinkle. It’s a community where you as a developer can share your knowledge with students in a more intimate way. Sprinkle your knowledge!

How I built my site: I decided to use CSS Grids for the layout. The CSS grid is divided in 12 fraction so it’s easy to divide the page in columns.

What I learned during this project:  simple JS, HTTP BIN, the Dom, inputs and accordions.

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -1,30 +1,164 @@
...
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+
+  <!-- Using Meta Tags to Prevent Form Zooming -->
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0,user-scalable=0" />
```
Comment (Jsfrulle) [inline code/index.html:9]: this is cool! really good to know!

File: code/script.js [hunk 2]
```diff
@@ -1,9 +1,22 @@
...
+
+//This code make a FAQ question that is toggled - close itself when another question is opened.
+// For each .question the JS will loop (===) the code and when it sees the code "btn active" it will close the others (remove active).
+
 function toggle() {
-  this.classList.toggle("active")
+    document.querySelectorAll(".question").forEach((btn, i) => {
+     
+        if (btn === this) 
+            return;
```
Comment (Jsfrulle) [inline code/script.js:18]: love(!!!) this effect on the according! when the user open a new section the other collapses!!!

File: code/index.html [hunk 3]
```diff
@@ -1,30 +1,164 @@
...
+            <label for="fname">Name*</label><br>
+            <input type="text" id="fname" name="fname" placeholder="Name" name="name" required>
+
+            <br><label for="email">Email*</label><br>
+            <input type="email" id="email" name="email" placeholder="Email" name="email" required><br>
+
+            <label for="password">Password*</label><br>
+            <input type="password" id="password" name="password" placeholder="Password" name="password" required><br>
+            <br>
+          </div>
```
Comment (Jsfrulle) [inline code/index.html:54]: I love br-tags but to get less code in the HTML-doc I think flexbox can be used here to structure the element in the same way. Display flexbox, flex-direction: column; and maybe padding-bottom to get more space for the input-"password"-element

File: code/script.js [hunk 4]
```diff
@@ -1,9 +1,22 @@
...
+        btn.classList.remove("active");
+    });
+    this.classList.toggle("active")
```
Comment (Jsfrulle) [inline code/script.js:21]: structured read and well-written code - just look up for using br-tags when flexbox maybe is a better solution. I love that you fix the zooming thing for phone users with the meta-tag. I also like the extra effect on the according done with javascript(a section automatically closing when the user clicks on a new one). The page is also very good-looking!   
- [YES ]  Am I able to understand the code easily?
- [well-written code]  Does the code have lots of duplicates? Could it be tidied up?
- [ Works ]  Does it work or does anything look broken?
- [ YES ]  Does it follow the general and blue requirements?

File: code/index.html (hunk 1 above)
Comment (Rosanna86) [reply code/index.html:9]: Thank you!

File: code/script.js (hunk 2 above)
Comment (Rosanna86) [reply code/script.js:18]: :) Yes I googled the hell out of that one!

File: code/index.html (hunk 3 above)
Comment (Rosanna86) [reply code/index.html:54]: Thank you for input!

**Comment by Rosanna86** [comment]
Thank your for all the input Jessica!

**Pull request closed without merging**